ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -182,6 +182,23 @@
   @Default("115")
   public final int storeIndexPersistedEntryMinBytes;
 
+  /**
+   * Specifies the size threshold (as percentage of maximum size) of a chunk for converting the chunk to RO from RW
+   */
+  @Config(storeDataReadOnlySizeThresholdPercentageName)
+  @Default("95")
+  public final int storeDataReadOnlySizeThresholdPercentage;
```

Comment:
minor: what's the requirement for "data" in here?

Reply:
Was copying convention from "storeDataFlushIntervalSeconds" and "storeDataFlushDelaySeconds"

Reply:
In the case of those variables, it's actual data being flushed. Here is it store being marked RO/RW (data cannot be marked RO/RW). This config is more along the lines of something like `store.enable.hard.delete`

Reply:
Changing names to store.readonly.enable.size.threshold.percentage and store.readwrite.enable.size.threshold.percentage.delta

Reply:
minor nit: if the variable is going to be `storeReadOnlySizeThresholdPercentage`, then the config has to be `store.read.only.enable.size.threshold.percentage` (or the variable has to be  `storeReadonlySizeThresholdPercentage`)

Reply:
Added the '.' between "read" and "(only)|(write)"